ui/components/common/CommonToolbar.js

```diff
@@ -47,6 +49,8 @@ class CommonToolbar extends Component {
                 type="text"
                 placeholder="Filter by Name"
                 className={toolbarFilter}
+                value={get(this.props.filter, "name", "")}
+                style={{ height: "26px" }}
```

Comment:
This should removed or at least included in the css class.

Reply:
ok